scripts/OpdrachtFiveBonus/Opdracht-5-Bonus.js

```diff
@@ -0,0 +1,18 @@
+let userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+console.log(userInput);
+
+const choice = prompt(`Je koos voor ${userInput}. Over welke functie wil je meer weten?
+Voer een getal tussen de 0 en 4 in. 0: ${departments[userInput].jobs[0].title}
+1: ${departments[userInput].jobs[1].title} 2: ${departments[userInput].jobs[2].title}
+3: ${departments[userInput].jobs[3].title}`)
+
+
+//  ----------------- Poging om error melding goed weer te geven, nog niet gelukt. Tips? ----------------
+// if (userInput =! "marketing", "sales", "customer-service"){
+//     document.getElementById('error-message').textContent = "Ongeldige keuze. Probeer het opnieuw door de pagina te verversen"
+//
+// }
```

Comment:
de id=error message; word nergens gebruikt, wat word afgebeeld op het schrem is de:   
default:
        console.error(departmentDescriptionElement.textContent = "Ongeldige keuze. " +
            "Probeer het opnieuw door de pagina te verversen");